frontend/src/employees/components/Employees.tsx

```diff
@@ -0,0 +1,21 @@
+import {Employee} from "../model/Employee";
+
+type ShowEmployeesProps = {
+    employees: Employee[],
+}
+
+export default function Employees(props: ShowEmployeesProps) {
+
+
+    return (
+        <div>
+            <h2>Mitarbeiter</h2>
+            <ul>
+                {props.employees.map((employee) => <li>{employee.name}</li>)}
```

Comment:
you need to specify a key when mapping over a list (check your developer console in your browser :) )